redbot/cogs/mod/settings.py

```diff
@@ -106,6 +112,24 @@ async def mentionspam(self, ctx: commands.Context):
         Manage the automoderation settings for mentionspam.
         """
 
+    @mentionspam.command(name="strict")
+    @commands.guild_only()
+    async def mentionspam_strict(self, ctx: commands.Context, toggle: bool):
+        """
+        Sets settings to allow for duplicate or only different mentions.
+
+        If enabled all mentions will count including duplicated mentions.
+        If disabled only unique mentions will count.
+
+        Default is settings is False.
```

Comment:
this would look better `Default settings is False.` instead of `Default is settings is False.`

Reply:
```suggestion
        Default setting is False.
```

Reply:
Wording changed, ty.